backend/index.js

```diff
@@ -120,7 +120,7 @@ app.get('/alert/:id', async (req, res) => {
 app.get('/alert-by-user/:id', async (req, res) => {
 
     const id = req.params.id
-    const alerts = await alertManager.find({ author_id: id })
+    const alerts = await alertManager.find(Object.assign({}, { author_id: id }, req.query)  )
```

Comment:
You can use the more elegant syntax:

```javascript
await alertManager.find({
    ...req.query,
    author_id: id
})
```

It's the standard way of doing it in JS, using the destructuring operator (```...```)